src/main/java/com/uid2/optout/vertx/OptOutServiceVerticle.java

```diff
@@ -161,7 +171,7 @@ private Router createRouter() {
         router.route(REPLICATE_METHOD)
                 .handler(auth.handle(this::handleReplicate, Role.OPTOUT));
         router.route(REFRESH_METHOD)
-                .handler(auth.handle(attest.handle(this::handleRefresh), Role.OPERATOR));
+                .handler(auth.handle(attest.handle(this::handleRefresh, Role.OPERATOR), Role.OPERATOR));
```

Comment:
Is this really supposed to be the Operator's role? Not Optout?

Reply:
Yes, it really is. The operator calls this method. You can see the auth handler has the same role